### DISCUSSION TITLE
審音

### DISCUSSION THREAD

**Discussion by syimyuzya** [ayaka14732/rime-tupa#2] [General]
之前直接討論過的兩例：

- 「拼」（「併合」義，「拚」（捨棄）是別的字）定為滂母青韻缺少理據。目前有音同「并」（平聲）的處理法。
- 「怎」是「作麼（物）」合音，宋代韻書中為精母侵韻一等，是切韻音系外的音。目前尚無規範處理法，已知做法有比如拼作 ~~tsoin~~ tsoim 或直接寫 tsakmut 等

Edit: 2022-04-23 補兩筆

| 字  | 拼音 | 地位 | 說明 | 原表 |
| --- | ---- | ---- | ---- | ---- |
| 咩哶 | miaeq | 明三麻上 | 本字「哶」《玉篇》莫者切《集韻》母野切 | 僅有「咩」，音 miae |
| 掰 | peek | 幫耕入 | 「擘」白讀，不是不規則音 | peej (逆推不當), peek |

TODO 待補完

**Comment by untunt** [comment]
几个后起字需要修正：

|      | 拼音   | 地位       | 说明                                                         | 原表            |
| ---- | ------ | ---------- | ------------------------------------------------------------ | --------------- |
| 撂   | lewh   | 來開四蕭去 | 本字“撩”，[玄应《音义》“擲也”，《集韵》增又音力弔切](https://www.zhihu.com/question/266813369/answer/313798928)。中原官话也读如效摄，不能是入声来源 | lyak(误)        |
| 摔   | sruit  | 生合三眞入 | 《中原音韵》入声作上声，《字汇》山律切，桓台、利津读入声（张鸿魁《“甩”“摔”和“蟀”的读音问题》、曾良《“甩”字的形音义考》） | srui(误), sruit |
| 甩   | sruit  | 生合三眞入 | 晚晴才流行，之前写作“摔”                                     | srueq(误)       |
| 擤 | haengq | 曉開二庚上 | [《五音集韵》虎梗切](https://www.zhihu.com/question/492481354) | siaengq(误) |
| 跑   | phaewq | 滂二肴上   | 和《广韵》“跑”（baew）恐怕不是一个字。从粤语看是二等。phaewq《广韵》《集韵》为空位 | baew, boeuk    |
| 妞 | nruq | 孃三尤上 | ①《集韵》女九切“姓也，髙麗有之”。②北京话的“妞”在清代读上声，可能是“女”白读 | (无) |
| 嗲   | tiaeq  | 端開三麻上 | 本字不明。此地位《集韵》《四声等子》《五音集韵》亦有“哆”（丁寫反） | (无)            |

来自合音的字比较麻烦，之后再列

**Reply by syimyuzya** [reply]
「甩」原表音支合太可怕了😂（說起來十年來真的沒有廣東人出來對這音提過「甩」廣州唸「率」變音（lat1）這事麼（x

**Reply by ayaka14732** [reply]
做的時候看到了，但是 lat1 是次濁聲母 + 陰調，是不合法搭配，所以想了想好像也不能説明什麼

![](https://user-images.githubusercontent.com/68557794/158988075-18f81c88-afcb-4b52-b13b-5460bbc47f57.jpg)

**Reply by syimyuzya** [reply]
:joy: 配圖哈哈哈哈哈哈

**Comment by untunt** [comment]
象征性提供一些合音字：

|      | 折合音 | 折合地位     | 本字音    | 说明                                                         | 原表            |
| ---- | ------ | ---------- | ------------------------------------------------------------ | --------------- | ---- |
| 您   | nrimq  | 孃開三侵上 | nryq-monq, nryq-lawq-njin-kae | ①复数，“你懑 > 我们 > 恁/您”合音，金代出现。《中原音韵》上声。（复数后缀“懑”唐代用“伟、弭”，金元也用“每”；“懑”在唐诗入韵读上声。当取上声）②尊称，“你老人家 > 你老 > 你纳 > 您”合音，19 世纪出现（读阳平可能源于“你老”的变调，不应采取） | nrim(误) |
| 俺 | ngomq | 疑開一覃上 | ngaq-monq | “(我懑 >) 我们 > 俺”合音，南宋、金出现，也作“唵”。折合音考虑按照声旁（“唵”qomq）入覃韵。另《广韵》“俺”有 qyemh、qyomh 两音，与人称代词无关 | qomq(误), qyemh, qyomh |
| 咱   | dza | 從開二歌平 | dzih-kae | “自家”合音，北宋出现，宋词入家麻韵。《四声篇海》子葛切，疑为明代所增，不应是入声 | dzom |
| 喒, 偺, 咱 | dzom | 從開二覃平 | dzih-kae-monq | “自家懑”合音，元代出现，后与“咱”混同。折合音考虑按照声旁（“昝”tsomq，为“朁”俗字）入覃韵 | dzom |
| 怎 | _tsoimq_ | _精開一侵上_ | tsak-mut | “作物 > 怎”合音，晚唐五代出现 | tsryimq(误) |
| 咋   | tsaq | 精開一歌上 | tsak-maq | “作物 > 作摩/作麼 > 怎”合音 | (无) |
| 啥   | djiaeq | 常開三麻上 | djieq-maq | “是物 > 是摩/是麼 > 啥”合音。昌徐型方言读如章组而不是庄组。声调可随“是、甚”取上声 | sjiaeq(误) |
| 旮   | koeuk | 見二江入 | koeuk | “角落”滞古音。[《四声篇海》明正德本增加“𤰙𤰚，音葛剌”](https://www.zhihu.com/question/439885644)，此 -t 尾不可靠 | kae(误)  |
| 旯   | lak | 來開一唐入 | lak |                                                              | (无)     |
| 倆   | lyaq   | 來開三歌上 | lyangq-kah | “两个 > 倆”合音。lyangq 是“伎～”            | lyangq   |
| 仨 | sa | 心開一歌平 | sam-kah | “三个 > 仨”合音 | sam(误), sap(误) |

**Reply by ayaka14732** [reply]
「您」在粵語中直接讀「你」（因為粵語沒有 -ein 韻母）

（每個香港小學生都知道）

![](https://user-images.githubusercontent.com/68557794/159221723-dd84b77f-1d91-4a43-8000-792d249e0ee9.png)

**Comment by syimyuzya** [comment]
修碼表時又發現兩個明顯可校的（見一樓）w

**Comment by syimyuzya** [comment]
再列出一些原表中待考的（可能都不太好考的）：

有女增補：
- 4005「不」幫魂入（似乎是依《韻會》「逋沒切」）

後起字等：
- 「屄」幫A脂~~上~~平（可能很難考（
- 「椪碰」滂耕去
- 「爿」並山平
- 「臢」精開寒平
- 「虼」見痕入
- 「裉」溪痕去
- 「褪」透魂去
- 「躉」端魂上
- 「鍘」崇開山入
- 「闆」滂山上
- 「賠」並灰平（《字彙》「古無此字俗音裴」《正字通》「賠本作備」視為「備」平聲(但「備」古屬職部，中古在灰韻也是可能的 (?)

**Reply by untunt** [reply]
- 「不」（更早的例子是《五音集韻》博沒切）
- 「屄」写错了，是平声（它是A类，脂韵是空位，支韵是卑，定为脂韵没问题）
- 「爿」（《上古音系》爿“𤖭今字”。《集韻》𤖭“步還切，片也”，删韵。但不妨先以谐声定为山韵）
- 「臢」应为 -m 尾，但是无法确定覃谈，待考（“腌臜”今音 āzā 则来自咸摄入声）
- 「鍘」本字是“𨰉䥷”，山韵读法当删（一语双关了！）

之前我筛选的其他一些不在《广韵》涵盖音节里的字：（不全）

- 「丟」端幽平（《四聲篇海》丁羞切。本字可能是“盩”）
- 「䟕」初開二麻去
- 「撳」溪B侵去
- 「偌」日開三麻去
- 「擃」孃江上
- 「套」透豪去

和

- 「嫑」幫A宵去
- 「銨」影開寒上
- 「羥」溪開陽上
- 「倆」來開三麻上（“两个”合音。《合并字学集韵》梨賈切）
- 「旯」來開二麻平（应为入声）

**Reply by syimyuzya** [reply]
> * 「丟」端幽平（《四聲篇海》丁羞切。本字可能是“盩”）
> 
> * 「䟕」初開二麻去
> 
> * 「撳」溪B侵去
> 
> * 「偌」日開三麻去
> 
> * 「擃」孃江上
> 
> * 「套」透豪去

這些也都是有女全字表裡就已經有的，除了「丟」外都是《集韻》的（有女表裡還有反切w（而且因為《集韻》有所以我沒列😂

**Comment by untunt** [comment]
现在回看的话，“怎”如果想硬拼，拼成 tsoymq 比较好。非三等的 [i、ɨ、u] 可拼作 ei、oy、ou，正好相当于三等的 [e、ə、o] ie、yo、uo 的颠倒（因此 oy 没必要拼成 eoy）

**Comment by untunt** [comment]
“丟”应采用端**四**尤平 tu，践行锐音拼三等 u 归尤韵（而不是幽韵）的规则
